test/function/probability/kldivergence.test.js

```diff
@@ -0,0 +1,37 @@
+var assert = require('assert');
+var math = require('../../../index');
+
+
+describe('kldivergence', function(){
+    it('should return 0, cause distributions is equals', function(){
+        var q = [0.1,0.4,0.5,0.2];
+        assert.equal(math.kldivergence(q, q), 0);
+        assert.equal(math.kldivergence(math.matrix(q), q), 0);
+        assert.equal(math.kldivergence(q, math.matrix(q)), 0);
+        assert.equal(math.kldivergence(math.matrix(q), math.matrix(q)), 0);
+    });
+
+    it('should return distance between two distrubutions', function(){
+        var q = [0.5,0.6,0.7];
+        var p = [0.4,0.5,0.6];
+        assert.notEqual(math.kldivergence(q, p), 0);
```

Comment:
Is it enough to test whether the result is non-zero? Shouldn't you test what the value of the actual outcome is equal to? (Same at line 23)